### PR TITLE
Add more Depends calls for generated code

### PR DESCRIPTION
**Pull request by tst-rwildes**
Fixes issue #1657

### CODE REVIEW

File: earth_enterprise/src/fusion/autoingest/SConscript [hunk 1]
```diff
@@ -149,6 +149,8 @@ for plugin in plugins:
     env.Command('plugins/'+plugin+'Asset.cpp',
                 [ 'AssetGenPublicCpp.pl', 'AssetGen.pm', src ],
                 [ "${SOURCES[0]} ${SOURCES[2]} $TARGET" ])
+    env.Depends('plugins/'+plugin+'Asset.cpp', 'plugins/'+plugin+'Asset.h')
```
Comment (tst-lsavoie) [inline earth_enterprise/src/fusion/autoingest/SConscript]: I don't think the .cpp needs to depend on the .h. More likely the .os needs to depend on both the .cpp and .h. Same comment in the other file.

File: earth_enterprise/src/fusion/autoingest/sysman/SConscript [hunk 2]
```diff
@@ -45,6 +45,10 @@ for plugin in plugins:
     env.Command('plugins/'+plugin+'AssetD.cpp',
                 [ '../AssetGenDaemonCpp.pl', '../AssetGen.pm', src ],
                 [ "${SOURCES[0]} ${SOURCES[2]} $TARGET" ])
+    env.Depends('plugins/'+plugin+'AssetD.cpp','plugins/'+plugin+'AssetD.h')
+    env.Depends('plugins/'+plugin+'AssetD.os', 'plugins/'+plugin+'AssetD.cpp')
+
+env.Depends('../plugins/ProjectedMapProjectAsset.cpp', 'plugins/MapLayerAssetD.h')
```
Comment (tst-lsavoie) [inline earth_enterprise/src/fusion/autoingest/sysman/SConscript]: I think this should be a .os depending on a .h as well.

File: earth_enterprise/src/fusion/autoingest/SConscript (hunk 1 above)
Comment (tst-rwildes) [reply earth_enterprise/src/fusion/autoingest/SConscript]: yep that makes sense

File: earth_enterprise/src/fusion/autoingest/sysman/SConscript (hunk 2 above)
Comment (tst-rwildes) [reply earth_enterprise/src/fusion/autoingest/sysman/SConscript]: will do

**Pull request merged**